src/generate.js

```diff
@@ -2,7 +2,12 @@ import { createCanvas, loadImage } from 'canvas';
 import fs from 'fs';
 import chalk from 'chalk';
 import sha1 from 'sha1';
-import {
+import minimist from 'minimist';
+const args = minimist(process.argv.slice(2));
+const configPath = args.config + 'config.json';
```

Comment:
For what I understood, this forces the dev to use a config.json.
The usage `--config ''` is a bit strange. for me, I would prefer something `--config ./config.json`